src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 200) {
+    this.health = health;
+    this.name = 'Animal';
```

Comment:
[LOGIC ERROR] The name property should be assigned the name passed into the constructor, not the hardcoded string 'Animal'.